index.js

```diff
@@ -0,0 +1,96 @@
+
+let battle = [];
+
+// debugger;
+document.addEventListener("DOMContentLoaded",()=>{
+    let button = document.querySelector("#getPokemon");
+    button.addEventListener("click", ()=>{
+        document.querySelector("#openingAudio").play()
+        getPokemon("pokemonOne");
+        getPokemon("pokemonTwo");
+    })
+    let buttonTwo = document.querySelector("#battlePokemon");
+    buttonTwo.addEventListener("click", ()=>{
+        battleHistory(battle);
+    })
+    
+})
+
+
+//function that declares a random winner
+const battleHistory = () => {
+    let randomWinner = Math.floor(Math.random()*battle.length);
+    let winner = battle[randomWinner];
+    // debugger;
+    let div = document.querySelector("#rdmWinner");
+
+    //comment if you don't want a history of all the battles won
+
+    // div.innerHTML = ""
+    let ul = document.createElement("ul");
```

Comment:
You can declare the ul at the top so u don't have to redefine it every time you add a winner